### PR TITLE
Fixed an issue where opening the settings dialog would sometimes crash the app

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

**Review by gave92 — DISMISSED** [review]

**Comment by gave92** [comment]
Looks good even if I'm not sure how can that happen, do you have steps to reproduce this issue?

**Comment by yaichenbaum** [comment]
> Looks good. I'm not sure how that can happen though, do you have steps to reproduce this issue?

It happens when you quickly click on the setting button multiple times before the dialog opens.

File: Files/ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -146,11 +146,11 @@ private void DetectFontName()
 
         public static async void OpenSettings()
         {
-            if (!UIHelpers.IsAnyContentDialogOpen())
+            if (UIHelpers.IsAnyContentDialogOpen())
```
Comment (gave92) [inline Files/ViewModels/MainViewModel.cs:149]: Mmmm it looked correct before.

Comment (yaichenbaum) [reply Files/ViewModels/MainViewModel.cs:149]: It was the opposite of what I wanted 😂
We only want to open the settings dialog when there aren't any other dialogs open.

Comment (yaichenbaum) [reply Files/ViewModels/MainViewModel.cs]: It doesn't seem to have worked to fix the issue 🤔

**Review by gave92 — APPROVED** [review]
Right the first time ;)

**Pull request merged**